spec/functions/nested_values_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'nested_values' do
+  # please note that these tests are examples only
+  # you will need to replace the params and return value
+  # with your expectations
```

Comment:
These comments should not have made it in.

Reply:
addressed in https://github.com/puppetlabs/puppetlabs-stdlib/pull/1363.